installing/acs-installation-platforms.adoc

```diff
@@ -1,14 +1,81 @@
 :_content-type: ASSEMBLY
 [id="acs-installation-platforms"]
-= Installation platforms and methods
+= Supported platforms and installation methods for RHACS
 include::modules/common-attributes.adoc[]
 :context: acs-installation-platforms
-
 toc::[]
 
 [role="_abstract"]
-{product-title} and {product-title-managed} are supported on various platforms, such as Red Hat OpenShift and Kubernetes platforms. 
+{product-title} ({product-title-short}) and {product-title-managed} ({product-title-managed-short}) are supported on various platforms, such as {osp} and Kubernetes platforms. 
+
+[id="install-platforms-methods_{context}"]
+== Installation methods for different platforms
+
+[role="_abstract"]
+You can perform different types of installations on different platforms.
+
+[NOTE]
+====
+Not all installation methods are supported for all platforms.
+====
+
+.Supported platforms and recommended installation methods
+[%autowidth]
+|===
+|*Platform type footnote:platform[The availability of support for each platform depends on the overarching lifecycle of the platform and the end-of-life date.]*|*Platform*|*Supported for Central*|*Supported for Secured Clusters*|*Supported installation methods*|*Installation steps*
```

Comment:
**question:** @agantony Can you please confirm the usage of `foornote` with OCP docs team ? I find it looks neat but I don't like them placed at the end of page. 
1. We can either remove footnotes and use superscript text with links to keep them both together. 
2. Or maybe move around the sections, so that Browser support and Architecture support section come before the supported platform section, that way footnotes stay with the table.